tests/test_srcset.py

```diff
@@ -96,6 +96,24 @@ def test_create_srcset_with_widths():
     assert expected == actual
 
 
+def test_create_srcset_with_variable_qualities():
```

Comment:
This test is good but I think we should also allow the user to have less than five srcset ratio descriptors if that's what they explicitly call for. Could we add a test for the following and ensure that it works?

```python
 ub = imgix.UrlBuilder(DOMAIN, include_library_param=False)
client.create_srcset("image.jpg", { "w": 100 }, { "variable_qualities": {1: 100, 2: 90, 3: 80} })

# https://testing.imgix.net/image.jpg?dpr=1&q=100&w=100 1x,
# https://testing.imgix.net/image.jpg?dpr=2&q=90&w=100 2x,
# https://testing.imgix.net/image.jpg?dpr=3&q=80&w=100 3x,
```

Reply:
Doesn't this relate to issue #101?

Reply:
I'm not sure I understand your question @ashutoshkrris.
The current test case we have is good but I think we should also write a test, and implement any changes for, the [use case](https://github.com/imgix/imgix-python/pull/102#discussion_r1018671654) I mentioned previously.